### ISSUE TITLE
Django, Caching and GET parameters

### ISSUE THREAD

**Issue by guettli** [ulope/django-reversetag#2]
From this project github page:

> If you're using Django's Cache Middleware it completely skips caching for pages with GET parameters!

AFAIK this is not true. This issue was solved long ago:

https://code.djangoproject.com/ticket/4992

**Comment by ulope** [comment]
Yeah, that readme was written [a long time ago](https://github.com/ulope/django-reversetag/commit/3d00f03e2162c62de78e28c08f650364fcc61d26) as well :)

This repo hasn't been updated in nearly 5 years. Since the `url` tag has also been fixed to accept variables long ago I personally have had little need for this. 

Partial resolving is still a nice feature but I found that I've used it less in practice than I initially thought.

In short: This project is all but abandoned. I'll put a notice to that effect into the readme (when I get around to it :))

**Comment by guettli** [comment]
The README was not changed. I guess it still gets read by some developer per day. 

Why did you close the issue. Do you think it is resolved?
